### PR TITLE
Do not use Show More on tag pages

### PR DESCRIPTION
**Pull request by robertberry-zz**
It doesn't make sense to hide things behind show more on tag pages, as the stories are chronologically ordered, not by terms of importance. This leads sometimes to the most important story on a day not being displayed by default, which is silly.

We never show more than 35 stories per page on tag pages, so this never makes pages massive either in terms of DOM nodes or scroll length.

You do sometimes get ugly containers, due to the mix of four- & three-column items, so we should possibly (in a separate pull request) make a 4-column show more for these cases. CC @sammorrisdesign 

screen shot 2015-02-12 at 11 30 28

I would like to get this out ASAP, though, as it fixes a current PROD bug where you can't see stories behind Show More.

### CODE REVIEW

**Comment by phamann** [comment]
:+1: nice.

**Comment by gklopper** [comment]
#BERRIED :+1:

File: common/app/views/fragments/containers/facia_cards/standardContainer.scala.html [hunk 1]
```diff
@@ -10,8 +10,8 @@
     <div class="@RenderClasses(Map(
         ("fc-container__body", true),
         ("fc-container--rolled-up-hide", true),
-        ("fc-show-more--hidden", true),
-        ("js-container--fc-show-more", true),
+        ("fc-show-more--hidden", containerDefinition.useShowMore && containerDefinition.containerLayout.exists(_.hasShowMore)),
```
Comment (johnduffell) [inline common/app/views/fragments/containers/facia_cards/standardContainer.scala.html]: consider to put these into a val as they look like they're identical?

Comment (gklopper) [inline common/app/views/fragments/containers/facia_cards/standardContainer.scala.html]: :+1: - especially if this is used in other places too.

**Comment by michaelwmcnamara** [comment]
Rob has confirmed looks ok on browsers. I checked a few iOS and Android mobiles and they look ok too.
+1

**Pull request merged**